docs/readmes/general/cloudstrapper_install.md

```diff
@@ -107,7 +107,7 @@ varFirstInstall: "false"
     - varBuildType: Using either ‘community’ or ‘custom’ binaries
     - varFirstInstall: Indicating if this is the first install of any kind of Magma or not, to skip some of the default, shared roles created
 
-- First change your directory to ~/magma-dev/magma/experimental/cloudstrapper/playbooks. Next, run the playbook to setup Orchestrator the deployment:
+- First change your directory to ~/magma-dev/magma/experimental/cloudstrapper/playbooks. Next, run the playbook to setup Orchestrator deployment:
```

Comment:
How about:
First change your directory to `~/magma-dev/magma/experimental/cloudstrapper/playbooks`. Next, run the playbook to setup the Orchestrator deployment:

Reply:
Sure! @Neudrino